build.gradle

```diff
@@ -122,6 +122,9 @@ test {
   useJUnit()
 }
 
+sourceCompatibility = '1.8'
```

Comment:
Modify copyright notice to reflect 2021

Reply:
> Is that a requirement?

No it's not, I add it because when I imported the project and build it, I faced a compatibility java version issue (java 55 on my computer vs java 52). The issue is visible when you import it on the warp10 instance. So I added this configuration to avoid futur contributor had this issue. But it's not "vital".
If you prefer I remove it.